### PR TITLE
[PoC] Add tests for dialog and detail.

### PR DESCRIPTION
**Pull request by thealmightyone**
I ignored eslint warnings for Table.jsx file.

### CODE REVIEW

File: src/DetailView/DetailActions.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export function fetchData(url, plural, cancelToken, polling) {
   return (dispatch, getState) => {
     const state = getState();
     const headers = {
+
```
Comment (stepek) [inline src/DetailView/DetailActions.js]: Please remove this empty line

File: src/Dialog/DialogActions.js [hunk 2]
```diff
@@ -16,15 +19,13 @@ function fetchError(data) {
 }
 
 function toLocalSchema(schema, state) {
-  // Convert dict in schema to array for form generation
-  // In json schema, we can't type dict element, so gohan
-  // extend json schema using items property for object.
-  // If object type has items property, items is considered to
-  // schema for object of dict.
-  // We will transform schema here for jsonform lib.
+
```
Comment (stepek) [inline src/Dialog/DialogActions.js]: Why you remove this comments ?

Comment (thealmightyone) [inline src/Dialog/DialogActions.js]: aren't those prehistoric

File: src/Dialog/DialogActions.js [hunk 3]
```diff
@@ -3,7 +3,10 @@ import {FETCH_SUCCESS, FETCH_FAILURE, CLEAR_DATA} from './DialogActionTypes';
 
 function fetchSuccess(data) {
   return dispatch => {
-    dispatch({data, type: FETCH_SUCCESS});
+    dispatch({
```
Comment (stepek) [inline src/Dialog/DialogActions.js]: Please restore this file.

**Comment by thealmightyone** [comment]
Reverted dialogActions.

**Pull request merged**